data/tutorials/lg_08_error_handling.md

```diff
@@ -196,6 +766,25 @@ match Sys.os_type with
 | _ -> failwith "this system is not supported"
 ```
 
-Note a couple of extra pattern matching features in this example too. A
-so-called "range pattern" is used to match either `"Unix"` or
-`"Cygwin"`, and the special `_` pattern which matches "anything else".
+It is right to use `failwith` because using `assert` would be incorrect. Here is
+the dual example:
+```ocaml
+function x when true -> () | _ -> assert false
+```
+Here, it wouldn't be beneficial to use `failwith` since it requires the compiler
+to be bugged or the system to be corrupted for second code path to be executed.
+Breakage of the language semantics qualifies as extraordinary circumstances; it
+is catastrophic.
```

Comment:
```suggestion
Here, it wouldn't be beneficial to use `failwith` because it requires a corrupted system or
for the compiler to be bugged for the second code path to be executed.
Breakage of the language semantics qualifies as extraordinary circumstances. It
is catastrophic!
```
Tried to reduce the repetitive to be... by taking one out. If it doesn't mean the same thing, feel free to reject it, but I'd still change "since" to "because."